src/pages/index.js

```diff
@@ -137,23 +303,28 @@ function handleProfileAddSubmit (formValue) {
 
 //Edit Form Events
 editProfile.addEventListener("click", () => {
+    editPostLoading.resetMessageEdit();
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.